it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -920,9 +920,108 @@ class PersistenceConnectorSpec
     }
   }
 
-  private def generateDepartureResponse(departureId: MovementId) =
-    DepartureResponse(
-      _id = departureId,
+  "GET /traders/movements/arrivals" - {
```

Comment:
endpoint with this should be GET /traders/${eoriNumber.value}/movements/arrivals?

Reply:
No, eoriNumber is passed as a query parameter in the api

Reply:
yes agree. But this is connector and  basically written to call the transit-movements api  endpoint /traders/:EORI/movements/arrivals . I am fine with both. Just asked if change is required.

Reply:
Oh yeah sorry you're right